Cargo.toml

```diff
@@ -374,11 +374,13 @@ which = { version = "4", default-features = false }
 wit-bindgen = { version = "0.32", default-features = false }
 wit-bindgen-core = { version = "0.32", default-features = false }
 wit-bindgen-go = { version = "0.32", default-features = false }
-wit-bindgen-wrpc = { version = "0.7", default-features = false }
+wit-bindgen-wrpc = { version = "0.8", default-features = false }
 wit-component = { version = "0.219", default-features = false }
 wit-parser = { version = "0.219", default-features = false }
-wrpc-interface-blobstore = { version = "0.19", default-features = false }
-wrpc-interface-http = { version = "0.29", default-features = false }
-wrpc-runtime-wasmtime = { version = "0.23", default-features = false }
-wrpc-transport = { version = "0.27", default-features = false }
-wrpc-transport-nats = { version = "0.24.1", default-features = false }
+wrpc-interface-blobstore = { version = "0.20", default-features = false }
+wrpc-interface-http = { version = "0.30", default-features = false }
+wrpc-runtime-wasmtime = { version = "0.24", default-features = false }
+wrpc-transport = { version = "0.28", default-features = false }
+wrpc-transport-nats = { version = "0.26", default-features = false, features = [
```

Comment:
I saw you tested with old/new host and old/new provider, but just to be certain, this new transport version is backwards compatible?

Reply:
yes